pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -43,6 +43,10 @@
 public class BrokerResponseNative implements BrokerResponse {
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  public static final BrokerResponseNative EMPTY_RESULT = new BrokerResponseNative();
```

Comment:
have a static method BrokerResponseNative.emptyResponse(). Makes it easy to set some defaults